### PR TITLE
[Components with more than 1 service] LPS-179348 Refactor ScreenNavigationCategory components from Commerce modules part 2

### PR DESCRIPTION
**Pull request by daiazius**

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:startup

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/commerce/commerce-notification-web/src/main/java/com/liferay/commerce/notification/web/internal/frontend/taglib/servlet/taglib/CommerceChannelNotificationsQueueScreenNavigationCategory.java [hunk 1]
```diff
@@ -43,26 +29,17 @@
...
+	service = ScreenNavigationCategory.class
 )
 public class CommerceChannelNotificationsQueueScreenNavigationCategory
-	implements ScreenNavigationCategory,
-			   ScreenNavigationEntry<CommerceChannel> {
+	implements ScreenNavigationCategory {
 
 	@Override
 	public String getCategoryKey() {
 		return "notification-queue";
```
[the remark this answers is not in the document]
Comment (rafaprax) [reply modules/apps/commerce/commerce-notification-web/src/main/java/com/liferay/commerce/notification/web/internal/frontend/taglib/servlet/taglib/CommerceChannelNotificationsQueueScreenNavigationCategory.java:40]: Hey @daiazius, thanks for your findings. 

Hey @tinatian as there is no pattern covering the constants used in getCategoryKey() and getEntryKey() methods, can we leave it as it is now? Thx

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by daiazius** [comment]
ci:test:startup

**Comment by Kvale1733** [comment]
@daiazius No regressions.

**Review by rafaprax — APPROVED** [review]

**Comment by daiazius** [comment]
Hey, I just changed the commit messages, no changes in the code.

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
Hey all, the only difference here is the baseline commit. So, the test results are still valid. Thx

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by daiazius** [comment]
ci:test:startup

**Review by rafaprax — APPROVED** [review]

**Comment by zhangyan0701** [comment]
ci:test:commerce

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by daiazius** [comment]
ci:test:startup

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
@daiazius regression found:

1. Create a minium site called "minium"
2. Go to applications menu > commerce > channels 
3. click in the site minium
4. Go to Notification Queue and Notification templates

Actual result:
any information is displayed and an error appears in the log:
```
2023-04-05 18:05:41.985 ERROR [http-nio-8080-exec-9][IncludeTag:128] Current URL /group/minium/~/control_panel/manage?p_p_id=com_liferay_commerce_channel_web_internal_portlet_CommerceChannelsPortlet&p_p_lifecycle=0&p_p_state=maximized&p_p_mode=view&_com_liferay_commerce_channel_web_internal_portlet_CommerceChannelsPortlet_mvcRenderCommandName=%2Fcommerce_channels%2Fedit_commerce_channel&_com_liferay_commerce_channel_web_internal_portlet_CommerceChannelsPortlet_screenNavigationCategoryKey=notification-queue&_com_liferay_commerce_channel_web_internal_portlet_CommerceChannelsPortlet_backURL=http%3A%2F%2Flocalhost%3A8080%2Fgroup%2Fminium%2F%7E%2Fcontrol_panel%2Fmanage%3Fp_p_id%3Dcom_liferay_commerce_channel_web_internal_portlet_CommerceChannelsPortlet%26p_p_lifecycle%3D0%26p_p_state%3Dmaximized%26p_p_auth%3DmjifREwh&_com_liferay_commerce_channel_web_internal_portlet_CommerceChannelsPortlet_commerceChannelId=44310&p_p_auth=mjifREwh generates exception: java.lang.NullPointerException
java.lang.NullPointerException: null
	at com.liferay.commerce.notification.web.internal.frontend.taglib.servlet.taglib.CommerceChannelNotificationsQueueScreenNavigationCategory.getLabel(CommerceChannelNotificationsQueueScreenNavigationCategory.java:50) ~[?:?]
	at org.apache.jsp.screen_005fnavigation.page_jsp._jspService(page_jsp:405) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111) ~[jasper.jar:9.0.73]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:596) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:411) ~[jasper.jar:9.0.73]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:473) ~[jasper.jar:9.0.73]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:377) ~[jasper.jar:9.0.73]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:596) ~[servlet-api.jar:4.0.FR]

```

Expected result:
Informations of Notification Queue and notification templates should display without errors.

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
@daiazius Please see https://github.com/liferay-core-infra/liferay-portal/pull/4141, fix the issue and resend.

**Pull request closed without merging**